.github/workflows/observer-e2e.yml

```diff
@@ -0,0 +1,108 @@
+name: Observer E2E
+
+on:
+  push:
+    branches:
+      - master
+      - 'auto-cadence-upgrade/**'
+      - staging
+      - trying
+      - 'feature/**'
+      - 'v[0-9]+.[0-9]+'
+  pull_request:
+    branches:
+      - master
+      - 'auto-cadence-upgrade/**'
+      - 'feature/**'
+      - 'v[0-9]+.[0-9]+'
+
+jobs:
+  golangci:
+    name: Lint1.45
+    runs-on: ubuntu-latest
+    steps:
+    - name: Setup Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.17
+    - name: Checkout repo
+      uses: actions/checkout@v2
+    - name: Download golangci-lint
+      run: bash -c 'curl -L https://github.com/golangci/golangci-lint/releases/download/v1.45.0/golangci-lint-1.45.0-linux-amd64.tar.gz | tar -xzC /opt'
+    - name: Run golangci-lint
+      run: bash -c '(/opt/golangci-lint-1.45.0-linux-amd64/golangci-lint run | sort >/tmp/lint.txt) && cat /tmp/lint.txt && (md5sum </tmp/lint.txt >.github/workflows/lintmd5.txt) && md5sum -c .github/workflows/lintmd5.txt </tmp/lint.txt'
+
+  integration-test:
+    name: Integration Tests
+    strategy:
+      fail-fast: false
+      matrix:
+        go-version:
+          - 1.16
+          - 1.17
+    runs-on: ubuntu-latest
+    steps:
+    - name: Setup Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{ matrix.go-version}}
+    - name: Checkout repo
+      uses: actions/checkout@v2
+    - name: Build relic
+      run: make crypto_setup_gopath
+    - name: Docker build
+      run: make docker-build-flow
+    - name: Run Observer End To End
+      run: bash -c 'cd integration/localnet/ && make -e OBSERVER=1 init && make start && (docker ps | grep observer) && make stop'
```

Comment:
This seems like a duplicate test of `flow-grpc-test`.

Reply:
I removed the file and I merged everything that we discussed necessary to ci.yml